src/support/timestamp.c

```diff
@@ -495,79 +510,57 @@ __wt_time_value_validate(
         WT_TIME_VALIDATE_RET(session,
           "value time window has a durable start time after its durable stop time; time window %s",
           __wt_time_window_to_string(tw, time_string[0]));
-
-    if (tw->prepare && F_ISSET(S2C(session), WT_CONN_PRESERVE_PREPARED)) {
-        if (!WT_TIME_WINDOW_HAS_START_PREPARE(tw) && !WT_TIME_WINDOW_HAS_STOP_PREPARE(tw)) {
+    /* Validate that if start prepare_ts is set, start_prepared_id must be set */
```

Comment:
This is only true for preserve prepared?

Reply:
moved the comment